SourceX/controls/plrctrls.h

```diff
@@ -5,6 +5,11 @@
 
 namespace dvl {
 
+typedef enum belt_item_type {

```

Comment:
nit: Use `enum class BeltItemType`, e.g.:

```
enum class BeltItemType {
  HEALTH_POTION,
  MANA_POTION
};
```

Reply:
why only hp/mana pots? what about rejuv pots/scrolls? :D

Reply:
These are for quick use potion buttons, L2/R2.
It will use a rejuv potion if there is no health/mana potions in belt

Reply:
or heal scrolls ;)

Reply:
All other enums are actually named in this fasion.